modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBAtlasLocalContainerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.testcontainers.containers;
+
+import org.junit.Test;
+
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.awaitility.Awaitility.await;
+
+public class MongoDBAtlasLocalContainerTest {
+
+    @Test
+    public void getConnectionString() {
+        try (
+            MongoDBAtlasLocalContainer container = new MongoDBAtlasLocalContainer(
+                MongoDBAtlasLocalContainer.DEFAULT_IMAGE_NAME.withTag("7.0.9")
+            )
+        ) {
+            container.start();
+            String connectionString = container.getConnectionString();
+            assertThat(connectionString).isNotNull();
+            assertThat(connectionString).startsWith("mongodb://");
+            assertThat(connectionString).isEqualTo(String.format("mongodb://localhost:%d/?directConnection=true", container.getFirstMappedPort()));
+        }
+    }
+
+    @Test
+    public void createAtlasIndexAndSearchIt() throws Exception {
+        try (
+            // creatingAtlasLocalContainer {
+            MongoDBAtlasLocalContainer atlasLocalContainer = new MongoDBAtlasLocalContainer(
+                MongoDBAtlasLocalContainer.DEFAULT_IMAGE_NAME.withTag("7.0.9")
+            );
+            // }
+        ) {
+            // startingAtlasLocalContainer {
+            atlasLocalContainer.start();
+            // }
+
+            // getConnectionStringAtlasLocalContainer {
+            String connectionString = atlasLocalContainer.getConnectionString();
+            // }
+
+            try (
+                AtlasLocalDataAccess atlasLocalDataAccess = new AtlasLocalDataAccess(connectionString, "test", "test")
+            ) {
+                atlasLocalDataAccess.initAtlasSearchIndex();
+
+                // writeAndReadBack {
+                atlasLocalDataAccess.insertData(new AtlasLocalDataAccess.TestData("tests", 123, true));
+
+                //Wait for Atlas Search to index the data (Atlas Search is eventually consistent)
+                await()
+                    .atMost(5, TimeUnit.SECONDS)
+                    .pollInterval(10, TimeUnit.MILLISECONDS)
+                    .pollInSameThread()
+                    .until(
+                        () -> atlasLocalDataAccess.findAtlasSearch("test"),
+                        Objects::nonNull);
+                // }
```

Comment:
`AtlasLocalDataAccess.TestData` is an class used only for testing. So, let's avoid showing it in the docs

Reply:
done - actually I already removed from the docs, I've now also removed the docs comments here too